srcpkgs/unixodbc/template

```diff
@@ -1,16 +1,21 @@
 # Template file for 'unixodbc'
 pkgname=unixodbc
-version=2.3.9
+version=2.3.11
 revision=1
 build_style=gnu-configure
 makedepends="libltdl-devel readline-devel"
 conf_files="/etc/odbcinst.ini /etc/odbc.ini"
 short_desc="Basic ODBC tools"
 maintainer="Orphaned <orphan@voidlinux.org>"
 license="GPL-2.0-only, LGPL-2.1-or-later"
-homepage="http://www.unixodbc.org"
-distfiles="http://www.unixodbc.org/unixODBC-${version}.tar.gz"
-checksum=52833eac3d681c8b0c9a5a65f2ebd745b3a964f208fc748f977e44015a31b207
+homepage="https://www.unixodbc.org"
+distfiles="https://www.unixodbc.org/unixODBC-${version}.tar.gz"
+checksum=d9e55c8e7118347e3c66c87338856dad1516b490fb7c756c1562a2c267c73b5c
+
+post_install() {
+	# Conflicts with isql from firebird3.
+	mv ${DESTDIR}/usr/bin/isql{,-uo}
```

Comment:
`isql-uo` looks kind of weird, does anybody have any alternative names?

Reply:
> Hrm, popcorn shows 2 installed unixodbc packages and no firebird3. I wonder whether firebird's executable should be renamed instead.

Those two could have been me. :P

Anyway, I went ahead and renamed `firebird3`'s `isql` to `isql-fb`.